### PR TITLE
Bootstrapping fix

### PR DESCRIPTION
**Pull request by Hellblazer**
Simplify bootstrapping.  Simplify phantasm services and exposure as a dropwizard application.  Provide a new maven plugin for rolling back a database instead of dropping it.  Allow configuration of all plugins/tools from environment to simplify integration into platforms like Heroku.

### CODE REVIEW

File: loader-plugin/src/main/java/com/chiralbehaviors/CoRE/loader/plugin/DatabaseLoader.java [hunk 1]
```diff
@@ -30,30 +30,6 @@
 /**
  * @author hhildebrand
  * 
- */
-/*
- * Copyright (c) 2009, 2011 Hal Hildebrand, all rights reserved.
```
Comment (meleemistress) [inline loader-plugin/src/main/java/com/chiralbehaviors/CoRE/loader/plugin/DatabaseLoader.java]: Why are we getting rid of the license?

File: schema/src/main/resources/com/chiralbehaviors/CoRE/schema/core.xml [hunk 2]
```diff
@@ -5,21 +5,14 @@
     http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd"
 >
 
-    <!-- Tag so we can roll back the change of the db creation -->
+    <!-- Tag fo roll back of the schema creation -->
```
Comment (meleemistress) [inline schema/src/main/resources/com/chiralbehaviors/CoRE/schema/core.xml]: "for"

File: loader-plugin/src/main/java/com/chiralbehaviors/CoRE/loader/plugin/DatabaseLoader.java (hunk 1 above)
Comment (Hellblazer) [inline loader-plugin/src/main/java/com/chiralbehaviors/CoRE/loader/plugin/DatabaseLoader.java]: whoops.  deleted the wrong dup.  Will fix

Comment (Hellblazer) [inline loader-plugin/src/main/java/com/chiralbehaviors/CoRE/loader/plugin/DatabaseLoader.java]: Ah.  Note that the original had the Chiral Behaviors copywrite.  This was a copy/paste error that I deleted.

**Pull request merged**